key-controller.js

```diff
@@ -1,18 +1,15 @@
 /*!
  * Key Details.
  *
- * Copyright (c) 2012-2014 Digital Bazaar, Inc. All rights reserved.
+ * Copyright (c) 2012-2017 Digital Bazaar, Inc. All rights reserved.
  *
  * @author Manu Sporny
  * @author David I. Lehn
  * @author Dave Longley
  */
-define([], function() {
-
-'use strict';
-
 /* @ngInject */
-function factory($scope, brAlertService, brKeyService, brRefreshService) {
+export default function factory($scope, brAlertService, brKeyService,
```

Comment:
Adding another comment because I meant to request changes in the previous review and can't figure out how to edit it.